plugins/server/io.github.cotrin8672/line-webhook/versions.ktor.yaml

```diff
@@ -0,0 +1 @@
+"[2.0,)":ktor-line-webhook-plugin:==
```

Comment:
This `==` notation is only used for plugins that version in sync with Ktor (i.e. plugins inside the Ktor repository).

Please use this here instead:
```
"[2.0,)": ktor-line-webhook-plugin:1.2.+
```